packager/packager.cc

```diff
@@ -1282,9 +1314,40 @@ ast::ParsedFile validatePackage(core::Context ctx, ast::ParsedFile file) {
     if (!absPkg.exists()) {
         // We already produced an error on this package when producing its package info.
         // The correct course of action is to abort the transform.
+
```

Comment:
Stray line leftover from previous iterations?

Reply:
Got rid of this line.